Cargo.toml

```diff
@@ -59,6 +59,11 @@ path = "benches/hierarchy/mod.rs"
 name = "misc"
 path = "benches/misc/mod.rs"
 
+[[bench]]
+name = "deadlock_detector"
+harness = false
```

Comment:
what does it mean?

Reply:
Without it, cargo bench won't print benchmark result.